Sources/TuistKit/Commands/InitCommand.swift

```diff
@@ -23,8 +23,8 @@ struct InitCommand: ParsableCommand, HasTrackableParameters {
     static var analyticsDelegate: TrackableParametersDelegate?
 
     @Option(
-        help: "The platform (ios, tvos or macos) the product will be for (Default: ios)",
-        completion: .list(["ios", "tvos", "macos"])
+        help: "The platform (ios, tvos, realityos, watchos or macos) the product will be for (Default: ios)",
+        completion: .list(["ios", "tvos", "macos", "realityos", "watchos"])
```

Comment:
should we use `xros`?

Reply:
I feel like by that logic we should say appletvos instead of tvos, which doesn't make sense
Also I suspect we'll see xrOS turn into visionOS in code eventually. Then we'll end up with a breaking user-facing change or a confusing xrOS variant that most people won't recognize

Reply:
Oops, that should've read `visionos`... Fixed.